sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -155,6 +157,12 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       settingKey[Map[String, util.JSON]]("Additional option to include in the generated 'package.json'")
 
     /**
+      * Should we enable the libraries workflow?
+      */
+    val bundlingMode: SettingKey[BundlingMode] =
+      settingKey[BundlingMode]("What [[ScalaJSBundlerPlugin.autoImport.BundlingMode]] should we use?")
```

Comment:
Maybe it’s better to prefix the key with `webpack`: `webpackBundlingMode`. Also, I doubt that the reference to the Scaladoc will be interpreted by sbt, in the key description :)

Reply:
Fair enough. Pushed an update with this change.